src/_store/AppConfigProvider.js

```diff
@@ -23,9 +24,11 @@ export default class AppConfigProvider extends Component {
         language: PropTypes.string.isRequired,
         theme: PropTypes.string.isRequired,
     };
-
+    
     render() {
         const { children, language, theme } = this.props;
+        addLocaleData(localeData(locale(language)));
```

Comment:
Supposingly render should be pure, is there any justification for this side-effect?